Improve Nginx settings

kreativmonkey opened the pull request:
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/andi34/photobooth/blob/dev/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "x" next to an item)
- [ ] Documentation update
- [x] Bug fix
- [ ] New feature
- [ ] Other, please explain:

#### What changes did you make? (Give an overview)

- Add new parameter to **setup the php version**
- Add variable vor **nginx**.conf
- Add php-fpm startup to **lighttpd** settings
- Add php version settings to the **lighttpd** settup
- Add **enable nginx** service to persist on reboot
- Add **enable php-fpm** service to persist on reboot
- Add **enable lighttpd** to persist on reboot
- Fix **client_max_body_size** problems
- Fix (improve) **lighttpd** service startup
- Fix (improve) **nginx** service startup
- Rename **lighttpd-php-config** variable
- Rename the variable for the **site_config**

#### Is there anything you'd like reviewers to focus on?
After some testing i recommand to switch the default webserver to nginx becouse of the performance and flexibility.

kreativmonkey committed:
fix nginx config

- Adding the php-version to the installationscript
- Adding variable vor nginx.conf
- rename the variable for the site_config
- improve the systemctl startup of nginx
- persist nginx for restarts
- start and enable php-fpm
- fix client_max_body_size problems

andi34 commented:
Hey and thanks for your PR!
To switch by default, we would need to detect the PHP Version as e.g. Ubuntu 22.04 uses PHP8(.1?) by default which will fail with the config for 7.4. Any Idea how to Deal with this? Lighttpd also only checks for 7.3 and 7.4 ...

kreativmonkey commented:
That is why i explicitly use php7.4-fpm to install and activation. you can do that because this versions are currently supported. You don't need to use the newest versions in the repository ;-)

kreativmonkey committed:
Add lighttpd fixes

- Adding new parameter to setup the php version
- Adding php version settings to the lighttpd settup
- Improve lighttpd service startup
- Add php-fpm startup to lighttpd settings
- Enable php-fpm and lighttpd to persist on reboot
